website/siteConfig.js

```diff
@@ -2,10 +2,10 @@ const siteConfig = {
     title: 'All Contributors',
     tagline: '✨ Recognize all contributors, not just the ones who push code ✨',
     // For deploy
-    url: 'https://allcontributors.org',
-    baseUrl: '/',
+    url: 'https://leahwasser.com',
+    baseUrl: '/all-contributors/',
     projectName: 'all-contributors',
-    organizationName: 'all-contributors',
+    organizationName: 'lwasser',
```

Comment:
Ditto about leaving a TODO (especially useful for those who have TODO extensions that highlight those comments).